modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,200 @@ The properties for broker-wide throughput quota balancing are configured at the
 
 == Client throughput limits
 
-Similar to <<broker-wide-throughput-limits>> but for clients, Redpanda provides configurable throughput quotas that apply to an individual client or a group of clients.
+Redpanda provides configurable throughput quotas that apply to an individual client or a group of clients. You can apply a quota for an individual client based on an exact match with its `client_id`, or a group of clients based on IDs that start with a given prefix. 
+
+As of v24.2, client throughput quotas are compatible with the https://cwiki.apache.org/confluence/display/KAFKA/KIP-546%3A+Add+Client+Quota+APIs+to+the+Admin+Client[AlterClientQuotas and DescribeClientQuotas^] Kafka APIs, and are separate from quotas configured through cluster configuration in earlier Redpanda versions. The client throughput quotas no longer apply on a per-shard basis, and now limit the rates across a Redpanda broker's node. The quotas are neither shared nor balanced between brokers.
+
+Redpanda supports the following Kafka API-based quota types on clients:
+
+|===
+| Quota type | Description
+
+| `producer_byte_rate`
+| Limit throughput of produce requests
+
+| `consumer_byte_rate`
+| Limit throughput of fetch requests
+
+| `controller_mutation_rate`
+| Limit rate of topic mutation requests, including create, add, and delete partition, in number of partitions per second
+
+|===
+
+You can also apply a default quota for all other client requests that don't have a specific quota based on an exact match or `client_id` prefix. 
+
+It is possible to create conflicting quotas if you configure the same quotas through both the Kafka API and a cluster configuration. Redpanda resolves these conflicts by following an order of preference in finding a matching quota for a request:
 
-NOTE: The client throughput quotas limit the rates within each shard (logical CPU core) of a Redpanda broker's node. The quotas are neither shared nor balanced between shards and brokers.
+. Quota configured through the Kafka API for an exact match on `client_id`
+. Quota configured through the Kafka API for a prefix match on `client_id`
+. Quota configured through cluster configuration properties (`kafka_client_group_byte_rate_quota`, `kafka_client_group_fetch_byte_rate_quota`, xref:upgrade:deprecated/index.adoc[deprecated starting in v24.2]) for a prefix match on `client_id`
```

Comment:
```suggestion
. Quota configured through cluster configuration properties (`kafka_client_group_byte_rate_quota`, `kafka_client_group_fetch_byte_rate_quota`, xref:upgrade:deprecated/index.adoc[deprecated in v24.2]) for a prefix match on `client_id`
```

Reply:
There might be a better way to point out that steps 3 & 5 are deprecated. Maybe start the step with: Deprecated:? Or put just (deprecated) next to the properties and add a note at the end of the list saying that the steps are included for completeness, but since the properties in steps 3 & 5 are deprecated in 24.2, steps 3 and 5 aren't considered in 24.2+ deployments?

Reply:
I think they would still be considered if they happen to be set @micheleRP (those properties are [still editable in 24.2](https://github.com/redpanda-data/docs/pull/592#discussion_r1675655005)) -- @pgellert can correct me if I've misunderstood, but I'm not sure we can say that 3 and 5 won't be considered.

Reply:
They are still considered in v24.2. After the migration period, in v25.1, these cluster configs are going to be removed and those steps won't be considered anymore.